godot-macros/src/class/data_models/interface_trait_impl.rs

```diff
@@ -340,22 +273,19 @@ pub fn transform_trait_impl(original_impl: venial::Impl) -> ParseResult<TokenStr
 
     let tool_check = util::make_virtual_tool_check();
 
-    // Use 'match' as a way to only emit 'Some(...)' if the given cfg attrs allow.
-    // This permits users to conditionally remove virtual method impls from compilation while also removing their FFI
-    // glue which would otherwise make them visible to Godot even if not really implemented.
-    // Needs '#[allow(unreachable_patterns)]' to avoid warnings about the last match arm.
-    // Also requires '#[allow(clippy::match_single_binding)]' for similar reasons.
-    let register_fn = convert_to_match_expression_or_none(register_fn);
-    let create_fn = convert_to_match_expression_or_none(create_fn);
-    let recreate_fn = convert_to_match_expression_or_none(recreate_fn);
-    let to_string_fn = convert_to_match_expression_or_none(to_string_fn);
-    let on_notification_fn = convert_to_match_expression_or_none(on_notification_fn);
-    let get_property_fn = convert_to_match_expression_or_none(get_property_fn);
-    let set_property_fn = convert_to_match_expression_or_none(set_property_fn);
-    let get_property_list_fn = convert_to_match_expression_or_none(get_property_list_fn);
-    let free_property_list_fn = convert_to_match_expression_or_none(free_property_list_fn);
-    let property_get_revert_fn = convert_to_match_expression_or_none(property_get_revert_fn);
-    let property_can_revert_fn = convert_to_match_expression_or_none(property_can_revert_fn);
+    let modifiers = modifiers.into_iter().map(|(cfg_attrs, modifier)| {
+        quote! {
+            #(#cfg_attrs)*
+            { item = item.#modifier::<#class_name>(); }
+        }
+    });
+
+    let item_constructor = quote! { {
+            let mut item = #prv::ITraitImpl::new::<#class_name>(#docs);
+            #(#modifiers)*
+            item
+        }
+    };
```

Comment:
```suggestion
    let item_constructor = quote! {
        {
            let mut item = #prv::ITraitImpl::new::<#class_name>(#docs);
            #(#modifiers)*
            item
        }
    };
```

Reply:
I'm also not 100% sure if "modifiers" is such a good term here -- it's typically used to denote values (flags as in Ctrl/Shift/...).

Since the inserted code aren't modifiers themselves but rather _modifications_, maybe name it something like `item_modifications` or `item_updates`?

(This mostly applies to the 2nd variable declaration; the list of `(cfg_attrs, ident)` is probably ok with the existing name).